lib/index.ts

```diff
@@ -13,13 +13,19 @@ var XML2JS_OPTS = {
   normalize: true,
   mergeAttrs: true
 };
+interface ParsedOutput {
+  pomXml: string;
+  pomObject: object;
+}
+type ParseOptions=(Options & { filePath?: string; xmlContent?: string }) | null
+type ParseCallback=(e: Error | null, r?: ParsedOutput | null) => void;
 
 /**
  * Parses xml into javascript object by using a file path or an xml content.
  * @param {object} opt Is the option with the filePath or xmlContent and the optional format.
  * @return {object} The pom object along with the timers.
  */
-function parse(opt, callback) {
+function parse(opt: ParseOptions,callback: ParseCallback) {
```

Comment:
Add return type to this function.

Reply:
Earlier I avoided hardcoding return types to functions and let typescript resolve those on its own, but since it's required so added those now.